src/types/onyx/Policy.ts

```diff
@@ -141,12 +141,12 @@ type QBOConnectionData = {
     vendors: Vendor[];
 };
 
-type IntegrationEntityMap = 'NONE' | 'DEFAULT' | 'TAG' | 'REPORT_FIELD';
+type IntegrationEntityMap = 'NONE' | 'DEFAULT' | 'TAG' | 'REPORT_FIELD' | false | true;
```

Comment:
I propose we replace `true` and `false` with "IMPORTED" and "NOT IMPORTED", it would be more self-explanatory

We could also add those strings ('NONE', 'DEFAULT', 'TAG', 'REPORT_FIELD', 'IMPORTED', 'NOT IMPORTED') to CONST.js and then have something like `type IntegrationEntityMap = (typeof CONST.INTEGRATION_ENTITY_MAP_TYPES)[keyof typeof CONST.INTEGRATION_ENTITY_MAP_TYPES];`
We would also be able to use this const on qbo pages instead of using 'REPORT_FIELD' and 'NONE' as strings, it's easy to make a typo